src/programs/CMakeLists.txt

```diff
@@ -47,14 +44,18 @@ if( HAVE_GPU )
                                   fiat parkind_${prec}
                                   eccodes_f90 eccodes_memfs
                                   ${MPI_Fortran_LIBRARIES}
-                                  trans_gpu_static_${prec}
+                                  trans_gpu_static_CA_${prec}
                                   gpu
                                   OpenACC::OpenACC_Fortran 
                                   ${LAPACK_LIBRARIES}
                                   nvhpcwrapnvtx
                           )
-	ecbuild_add_executable(TARGET  driver-spectrans-CA-${prec}
-                             SOURCES driver-spectraltransform.F90
+      set_property( TARGET driver-spectrans-CA-${prec} PROPERTY CUDA_ARCHITECTURES 70 )
```

Comment:
CMake changes are just to make it work for me! I don't see myself as a ecbuild expert; and I think there have been changes in master.